docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
```

Comment:
```suggestion
Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons.
```

We can make it more explicit by already mentioning the props that allow me to add custom icons right away!

Also, removed the "You can use icons libraries" because that made me think we'd be using a library that's _not_ the Material Icons one, even though I now get the meaning!